### ISSUE TITLE
License

### ISSUE THREAD

**Issue by chnoma** [ErikMinekus/traktor-api-client#11]
Hi! I've been using this script for awhile now to do some pretty cool stuff, like semi-timecoded lighting and OBS screens showing track info, and I'd really like to start publishing some of those scripts and templates. It's really amazing what you can do with this.

That being said, a lot of these require modifications of this script particular to those applications, but I don't feel comfortable forking it without a specified license. Could you add one? Are you OK with forking?

Thanks so much for the great work on this

**Comment by ErikMinekus** [comment]
Sorry for the late response, I needed some time to decide on a license. I mostly care that people can include this library in their own projects and credit me for the work, so I went with the MIT license.

I'm ok with forking, and I'm also happy to accept PRs for features you think can be used for multiple applications. I would love to see how you are using this library, if you're willing to share.

**Comment by lautrip** [comment]
> ld you add on



> Hi! I've been using this script for awhile now to do some pretty cool stuff, like semi-timecoded lighting and OBS screens showing track info, and I'd really like to start publishing some of those scripts and templates. It's really amazing what you can do with this.
> 
> That being said, a lot of these require modifications of this script particular to those applications, but I don't feel comfortable forking it without a specified license. Could you add one? Are you OK with forking?
> 
> Thanks so much for the great work on this

I'm curious, is it possible to see some of that stuff you made?

**Comment by chnoma** [comment]
Oh I am super super late on this, but yeah!
me and some roommates do basement sets and stream to friends

sound warning for both of these

https://youtu.be/S_jFPBEskeU
here's a goofy mix I did -- the screen in the center is driven by a server receiving info through this extension

https://youtu.be/qb0joOxNBfE?t=11552
and here's a vid of a lighting and display setup we did for a halloween set this year

The traktor extension pretty much constantly sends to a server that everything else pulls from. The server also calculates phase so that we can do on-beat, off-beat lighting, allll of which is controlled from an open stage control page.

and excuse the mixing, we get very drunk
